src/main/java/com/app/bookstore/service/book/BookService.java

```diff
@@ -10,4 +10,8 @@ public interface BookService {
     BookDto findById(Long id);
 
     List<BookDto> findAll();
+
+    void deleteBookById(Long id);
+
+    BookDto updateBookById(Long id, CreateBookRequestDto createBookRequestDto);

```

Comment:
Also you will have only one update method, so we can just update.
```suggestion
    BookDto update(Long id, CreateBookRequestDto createBookRequestDto);
```